### PR TITLE
update dependency to fetch to support 0.1.4 and 0.1.4-beta300.0

### PR DESCRIPTION
**Pull request by vparpoil**
solves https://github.com/veliovgroup/meteor-cookies/issues/21

### CODE REVIEW

File: package.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ Package.onUse((api) => {
   api.versionsFrom(['1.4', '3.0-beta.0']);
```
Comment (dr-dimitru) [inline package.js:10]: @vparpoil shall we bump this line for the latest beta too?

**Review by dr-dimitru — COMMENTED** [review]
@vparpoil please see my question regarding bumping `api.versionsFrom()`

File: package.js (hunk 1 above)
Comment (vparpoil) [reply package.js:10]: Hi @dr-dimitru , From what I have seen elsewhere, the practice seems to use `'3.0-beta.0'` which makes this valid for all `3.0-beta.X` in my understanding 
[Here is an example for collection2](https://github.com/Meteor-Community-Packages/meteor-collection2/blob/master/package/collection2/package.js#L19)

Comment (dr-dimitru) [reply package.js:10]: @vparpoil perfect 👍

**Comment by dr-dimitru** [comment]
@vparpoil published as [`v2.8.1`](https://github.com/veliovgroup/meteor-cookies/releases/tag/2.8.1) thanks a lot for contribution 🙏

**Pull request merged**